codalab/rest/account.py

```diff
@@ -75,8 +76,23 @@ def do_signup():
     last_name = request.forms.get('last_name')
     password = request.forms.get('password')
     affiliation = request.forms.get('affiliation')
+    token = request.forms.get('token')
 
     errors = []
+
+    url = 'https://www.google.com/recaptcha/api/siteverify?secret={}&response={}'.format(
+        os.environ['CODALAB_RECAPTCHA_SECRET_KEY'], token
```

Comment:
Will this throw an error if `CODALAB_RECAPTCHA_SECRET_KEY` is not defined? Is that variable always defined (so we don't have to worry about this edge case)?

Reply:
Since the variable will either be the default key or overridden by the developer, I don't since it will ever be undefined.